Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -0,0 +1,234 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
+protocol ReaderModeStyleViewControllerDelegate {
+    func readerModeStyleViewController(readerModeStyleViewController: ReaderModeStyleViewController, didConfigureStyle style: ReaderModeStyle)
+}
+
+class ReaderModeStyleViewController: UIViewController {
+    var delegate: ReaderModeStyleViewControllerDelegate?
+    var readerModeStyle: ReaderModeStyle = DefaultReaderModeStyle
+
+    private let rowHeights = [72, 44, 44, 52]
+    private let width = 260
+
+    private var rows: [UIView] = []
+    private var fontTypeButtons: [FontTypeButton]!
+    private var fontSizeButtons: [FontSizeButton]!
+    private var themeButtons: [ThemeButton]!
+
+    override func viewDidLoad() {
+        view.backgroundColor = UIColor.grayColor()
+
+        // Our preferred content size has a fixed width and height based on the rows + padding
+
+        let height = rowHeights.reduce(0, combine: +) + (rowHeights.count - 1)
+        preferredContentSize = CGSize(width: width, height: height)
+
+        // Setup the rows. It is easier to setup a layout like this when we organize the buttons into rows.
+
+        for idx in 0...3 {
```

Comment:
It's nice that you're able to use a rows array to reuse logic in certain places, but it's not so nice that you have a bunch of magic indices everywhere. Would you object to something like this?
```
let fontTypeRow = UIView()
let fontSizeRow = UIView()
let themeRow = UIView()
let sliderRow = UIView()
let rows = [fontTypeRow, fontSizeRow, themeRow, sliderRow]
```

Then instead of <code>for idx in 0...3</code>, you would have <code>for row in rows</code>. And instead of things like <code>inRow: rows[0]</code>, you could do <code>inRow: fontTypeRow</code>. So things are more named this way, but you still have the benefits of your row array.